schedule/yast/autoyast/autoyast_cryptlvm+activate_existing.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: autoyast_cryptlvm+activate_existing
+description: >
+    Conduct autoyast installation activating encrypted partitions
+vars:
+  AUTOYAST: autoyast_sle15/autoyast_reuse-encrypted.xml
+  AY_INTERRUPT: '1'
+schedule:
+  - autoyast/prepare_profile
+  - installation/bootloader_start
+  - autoyast/installation
+  - console/validate_activation_encrypted_partition
+  - autoyast/installation
```

Comment:
to use twice a module as messy as this one is just calling for tricky maintenance of the scenario IMHO.